src/server/middleware/systraceMiddleware.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+import type { Middleware, $Request } from 'express';
+
+const fs = require('fs');
+const dedent = require('dedent');
+const logger = require('../../logger');
+
+const systraceMiddleware: Middleware = (req: $Request, res, next) => {
+  if (req.path !== '/systrace') {
```

Comment:
Not sure. From a developer point of view, it would make sense to use that pattern and abstract the path away. In case of RN-specific middleware, having this configurable doesn't make any sense given that it's hardcoded in React Native and should be tested in Haul to match these values.